### PR TITLE
Update pip tools

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Summary

Updates `pip-tools` to 6.1.0 and other requirement files after running `make requirements`

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Safety story

changes have been tested locally.
### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: requirements/dev-requirements.txt [hunk 1]
```diff
@@ -10,6 +10,8 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
```
Comment (biyeun) [inline requirements/dev-requirements.txt:13]: @AmitPhulera unfortunately this breaks the build as it's specific to mac os users. I generally manually remove it

File: requirements/prod-requirements.txt [hunk 2]
```diff
@@ -8,6 +8,8 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
+    # via ipython
```
Comment (millerdev) [inline requirements/prod-requirements.txt:12]:
```suggestion
```

File: requirements/dev-requirements.txt [hunk 3]
```diff
@@ -10,6 +10,8 @@ alembic==1.4.3
...
+    # via ipython
```
Comment (millerdev) [inline requirements/dev-requirements.txt:14]:
```suggestion
```

**Review by millerdev — COMMENTED** [review]
As @biyeun mentioned, the `appnope` dependencies should be removed. That has been an issue for a while, not something introduced with the pip-tools update.

Approved after they are removed.

File: requirements/dev-requirements.txt (hunk 1 above)
Comment (AmitPhulera) [reply requirements/dev-requirements.txt:13]: Ohh!! I did not know that. Thanks for the info. I will remove it.

**Comment by AmitPhulera** [comment]
Thanks, @biyeun, and @millerdev. I have removed `appnove` from requirements.

**Review by biyeun — APPROVED** [review]

**Review by millerdev — APPROVED** [review]

**Pull request merged**